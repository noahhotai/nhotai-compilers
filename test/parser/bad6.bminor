func1: function integer (x:integer, y:char) =
{
	print x;
	return 1
}