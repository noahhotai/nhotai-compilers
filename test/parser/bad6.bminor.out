IDENTIFIER     func1
COLON :
FUNCTION            function
INT            integer
LEFT PARENTHEIS     (
IDENTIFIER     x
COLON :
INT            integer
COMMA          ,
IDENTIFIER     y
COLON :
CHAR           char
RIGHT PARENTHESIS    )
ASSIGN         =
LEFT BRACE     {
PRINT          print
IDENTIFIER     x
SEMICOLON      ;
RETURN         return
INT LITERAL    0
RIGHT BRACE    }
parse error: syntax error
